src/Components/ServiceScene/ActionBarScene.tsx

```diff
@@ -39,6 +41,10 @@ export class ActionBarScene extends SceneObjectBase<ActionBarSceneState> {
       <Box paddingY={0}>
         <div className={styles.actions}>
           <Stack gap={1}>
+            {
+              // @ts-ignore appSidecar not yet in stable runtime
+              config.featureToggles.appSidecar && <ToolbarExtensionsRenderer serviceScene={serviceScene} />
```

Comment:
Not sure this needs to be behind a featureFlag. The code itself does not have any special tie to sidecar, just the reason for adding it now is to test it.